docs/Internal Systems/README.md

```diff
@@ -0,0 +1,3 @@
+# Internal Systems Documentation
+
```

Comment:
I would be good to create a contents page here that hyperlinks to the research that has been uploaded

Reply:
I have initiated this but the link are not yet finalized they can be once git has been updated to the files I have created in this PR